pykorm/meta.py

```diff
@@ -45,16 +47,26 @@ def _get_attributes(cls) -> Dict[str, fields.DataField]:
         return retl
 
     def __setattr__(self, item: str, value):
-        attrs = self._get_attributes()
-        if item in attrs:
-            attr = attrs[item]
-            current_val = getattr(self, item)
-
-            if attr.readonly and current_val is not None:
-                # We allow to set the attribute if it was not set before
-                if current_val != value:
-                    raise ReadOnlyException(f'{item} attribute is read_only !')
-
+        from .models import Nested
+        for attr_name, attr in self._get_attributes().items():
+            if item == attr_name:
+                current_val = getattr(self, item)
+                if attr.readonly and current_val is not None:
+                    # We allow to set the attribute if it was not set before
+                    if current_val != value:
+                        raise ReadOnlyException(f'{attr_name} attribute is read_only !')
+                if value is not None:
+                    if isinstance(attr, fields.ListField):
```

Comment:
This is to enable nested classes to pass dictionaries or arrays directly, instead of passing reference fields every time, I haven't found a way to put it in the __setattr__ of the DataField, if you can do it, you can help to change it